staff/gustavo-rojas/course-2/task-3/tgif-project/pro-congress-117-senate.json

```diff
@@ -0,0 +1,4912 @@
+{
```

Comment:
Duplicated file, please remove the one that you are not using.

Reply:
Done .